src/Components/Table.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from "react";
+import MUIDataTable from "mui-datatables";
+import { Grid } from "@mui/material";
+
+const columns = [
+	"id",
+	"msisdn",
+	"country",
+	"operator",
+	"regdate",
+	"routed",
+	"success",
+	"failure",
+	"error"
+];
+
+const options = {
+	filterType: "checkbox"
+};
+
+function App() {
+	const [data, setData] = useState([]);
+
+	useEffect(() => {
+		const fetchData = async () => {
+			try {
+				const response = await fetch(
+					"https://660549352ca9478ea17ff019.mockapi.io/gateway/api/table-data"
```

Comment:
Avoid hardcoding API links. Consider retrieving them from environment files. Refer to the link below for more information: https://create-react-app.dev/docs/adding-custom-environment-variables/